JAVA/src/main/java/it/bancaditalia/oss/sdmx/client/custom/ABS2.java

```diff
@@ -0,0 +1,32 @@
+package it.bancaditalia.oss.sdmx.client.custom;
+
+import it.bancaditalia.oss.sdmx.api.Dataflow;
+
+import it.bancaditalia.oss.sdmx.client.RestSdmxClient;
+import it.bancaditalia.oss.sdmx.exceptions.SdmxException;
+import it.bancaditalia.oss.sdmx.exceptions.SdmxExceptionFactory;
+import it.bancaditalia.oss.sdmx.parser.v21.Sdmx21Queries;
+import it.bancaditalia.oss.sdmx.util.RestQueryBuilder;
+
+import javax.net.ssl.SSLSocketFactory;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+/**
+ * ABS ReST 2.1 endpoint
+ */
+public class ABS2 extends RestSdmxClient {
```

Comment:
Hi Luca, thanks for this PR. There is no need to create an ABS2 custom provider: since the ABS2 is sdmx 2.1 compliant (at least for what concerns our connectors) you can simply add it in the [SDMXClientFactory ](https://github.com/amattioc/SDMX/blob/dba4db2765a9a9726e47f2fd64aaaf2f5f6ddd04/JAVA/src/main/java/it/bancaditalia/oss/sdmx/client/SDMXClientFactory.java#L103) class. You can check how the ILO provider is handled, for example:

`private static final String ILO_PROVIDER = "https://www.ilo.org/sdmx/rest";`
`....`
`addBuiltInProvider("ILO", ILO_PROVIDER, false, false, false, "International Labour Organization", false);`

Reply:
Sure. I had added ti as a standalone class while experimenting with it to retain some flexibility in case the new ABS endpoint turned out to be non-strictly 2.1-compliant.